src/arrow-util/src/builder.rs

```diff
@@ -327,11 +373,62 @@ impl ColBuilder {
                 builder.append_value(ts.timestamp_micros())
             }
             (ColBuilder::LargeBinaryBuilder(builder), Datum::Bytes(b)) => builder.append_value(b),
+            (ColBuilder::FixedSizeBinaryBuilder(builder), Datum::Uuid(val)) => {
+                builder.append_value(val.as_bytes())?
+            }
             (ColBuilder::StringBuilder(builder), Datum::String(s)) => builder.append_value(s),
+            (ColBuilder::LargeStringBuilder(builder), _)
+                if self.original_type == ScalarType::Jsonb =>
+            {
+                builder.append_value(JsonbRef::from_datum(datum).to_serde_json().to_string())
+            }
```

Comment:
Can we not just match on the `Datum` being a jsonb datum?

Reply:
There is no `Datum::jsonb` type! I learned that recently:
https://github.com/MaterializeInc/materialize/blob/09bc3c31644c0e8c67d260be1cbe1fdbbd9b692e/src/repr/src/scalar.rs#L1475-L1486

Reply:
omg how did I forget this!